psalm-baseline.xml

```diff
@@ -1474,6 +1475,56 @@
       <code>getAttribute</code>
     </UndefinedMethod>
   </file>
+  <file src="src/Reader/Extension/PodcastIndex/Entry.php">
+    <MixedInferredReturnType occurrences="3">
+      <code>null|stdClass</code>
```

Comment:
What's the right syntax here>

Reply:
I think psalm is explicitly complaining about our abuse of `stdClass` here. Can we use a well defined value object instead?

Reply:
> Can we use a well defined value object instead?

Unfortunately no. The whole type system of this component is a horror and if we introduce something new here, it won't make it any better. The only thing that remains is the consistency, even if it is poor. (See #30)
A complete rework is planned for version 3.

Reply:
What we could do is documenting with `@psalm-return object{foo: t, bar: t2, ...}` - that would be a decent start, and probably acceptable for the tool 🤔

Reply:
> that would be a decent start, and probably acceptable for the tool

Definitely!

Reply:
Apologies for taking so long to look at this. I tried to do exactly what you describe but that's not fixing everything. It's back to suggesting `null|stdClass` as the return type 👎